src/box/iproto_constants.h

```diff
@@ -179,6 +179,17 @@ enum iproto_key {
 	 * authentication method.
 	 */
 	IPROTO_AUTH_TYPE = 0x5b,
+	/**
+	 * Space name used instead of identifier (IPROTO_SPACE_ID) in CRUD
```

Comment:
Nit: Nowhere in the code we use the term "CRUD". We call these requests "DML". Using two terms for the same thing may look confusing. Better continue using "DML".

Reply:
Actually DQL (select, get) is not DML, but in terms of our codebase they are, which is confusing. Fixed.